### PR TITLE
Fix headaches when reading osu!droid code.

### PR DESCRIPTION
**Pull request by Reco1I**
Recreating PR with better commit structure.

This is a Intellij IDEA Code Inspection tool assisted reformat and cleanup which only affects the main package `ru.nsu.ccfit.zuev`.

### CODE REVIEW

**Comment by Reco1I** [comment]
👍🏻

File: src/ru/nsu/ccfit/zuev/osu/scoring/ScoringScene.java [hunk 1]
```diff
@@ -98,47 +97,66 @@ public void load(final StatisticV2 stat, final TrackInfo track,
...
 
         final TextureRegion hit100sr = ResourceManager.getInstance().getTexture("hit100");
-        final Sprite hit100s = new Sprite(Utils.toRes(10), Utils.toRes(130 + 92),
-                Utils.toRes(hit100sr.getWidth()), Utils.toRes(hit100sr.getHeight()), hit100sr);
-        hit100s.setPosition(Utils.toRes(70 - hit100s.getWidth() / 2 + x), Utils.toRes(130 + 92 - hit100s.getHeight() / 2 + y));
+        final int i8 = hit100sr.getHeight();
+        final int i9 = hit100sr.getWidth();
+        final Sprite hit100s = new Sprite(10, 130 + 92, i9, i8, hit100sr);
+        final float i22 = 130 + 92 - hit100s.getHeight() / 2 + y;
+        final float i23 = 70 - hit100s.getWidth() / 2 + x;
```
Comment (Rian8337) [inline src/ru/nsu/ccfit/zuev/osu/scoring/ScoringScene.java:119]: Well this is... something.

Comment (Reco1I) [reply src/ru/nsu/ccfit/zuev/osu/scoring/ScoringScene.java:119]: What the frick lol

**Comment by Reco1I** [comment]
I gonna close this because will be a mess to merge now.

Anyways there's some things that needs to be looked and eventually fixed/merged:

- Commit [6617c13](https://github.com/osudroid/osu-droid/pull/314/commits/6617c13baf4254a23ba6da942b7e1247c552e08c)
- Commit [149f0a7](https://github.com/osudroid/osu-droid/pull/314/commits/149f0a7a4bd28b0f671023c5527686fd50e3487e)
- Commits marked with `Bugs` tag (these are extremely important).
- Commits marked with `Performance` tag.
- Commits marked with `Java` tag.

**Comment by ammarasyad** [comment]
Noted. Some changes here may be incorporated to osu!droid in the future, accounting for changes not present in this PR.

**Pull request closed without merging**